interoperability-layer-openhim/package-metadata.json

```diff
@@ -6,7 +6,7 @@
   "type": "infrastructure",
   "dependencies": [],
   "environmentVariables": {
-    "STATEFUL_NODES": "single",
+    "NODE_MODE": "single",
     "MONGO_SET_COUNT": "1",
```

Comment:
We don't need MONGO_SET_COUNT anymore, right ?

Reply:
we still need it in the script initiate-replica-set.sh |this script needs to be refactored as well]